.github/workflows/nightly.yml

```diff
@@ -91,6 +90,33 @@ jobs:
           kubectl run --rm --attach test-http --image=galoisinc/cryptol-remote-api:test --image-pull-policy=Never --restart=Never -- http cra-http-cryptol-remote-api 8080
           kubectl run --rm --attach test-socket --image=galoisinc/cryptol-remote-api:test --image-pull-policy=Never --restart=Never -- socket cra-socket-cryptol-remote-api 8080
 
+  docker-cryptol-remote-api-portable:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+      - id: docker_meta
```

Comment:
Using this pattern enables use of both registry caching from buildx and the git labels from `build-push-action@v1`

Reply:
:+1: